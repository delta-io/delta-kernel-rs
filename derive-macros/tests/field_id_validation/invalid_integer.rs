use delta_kernel_derive::ToSchema;

#[derive(ToSchema)]
struct TestStruct {
    #[field_id = 18446744073709551616]  // This is u64::MAX + 1, too large for i64
    field: String,
}

fn main() {}
