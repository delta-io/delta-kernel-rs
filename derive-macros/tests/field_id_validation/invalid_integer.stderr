error: field_id error: Failed to parse integer: number too large to fit in target type
 --> tests/field_id_validation/invalid_integer.rs:5:18
  |
5 |     #[field_id = 18446744073709551616]  // This is u64::MAX + 1, too large for i64
  |                  ^^^^^^^^^^^^^^^^^^^^
